### ISSUE TITLE
Can this be used to dump ads to a local database?

### ISSUE THREAD

**Issue by FrodeMMeling** [tellnes/finn.no#1]
An example of usage would be nice.

**Comment by tellnes** [comment]
Yes, it should be possible to dump ads to a local database. But you need to write some code yourself.

As of now it is possible to search for ads and stream the results. The code will also let you request any document from the api and take care of authorization and caching.

Yes, some documentation would be nice. I will add that when I have some time.
